packages/react-api/src/Api.tsx

```diff
@@ -22,6 +22,7 @@ import addressDefaults from '@polkadot/util-crypto/address/defaults';
 
 import ApiContext from './ApiContext';
 import registry from './typeRegistry';
+import initSettings from '@polkadot/apps/initSettings';
```

Comment:
Would prefer the API not import from `@polkadot/apps` (The ends up being a circular dep when all is said and done)